cpp/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
 
 project(CUDA_DATAFRAME VERSION 0.7.2 LANGUAGES C CXX CUDA)
 
+if(NOT CMAKE_CUDA_COMPILER)
+  message(SEND_ERROR "CMake cannot locate a CUDA compiler")
```

Comment:
Should we `FATAL_ERROR` this instead?

Reply:
No, why? Let's collect all the errors. We don't need the compiler to do Makefile generation.